code/controllers/ContentController.php

```diff
@@ -363,21 +363,25 @@ public function SiteConfig() {
 	 * Inspects the associated {@link dataRecord} for a {@link SiteTree->Locale} value if present,
 	 * and falls back to {@link Translatable::get_current_locale()} or {@link i18n::default_locale()},
 	 * depending if Translatable is enabled.
-	 * 
+	 *
 	 * Suitable for insertion into lang= and xml:lang=
 	 * attributes in HTML or XHTML output.
-	 * 
+	 *
 	 * @return string
 	 */
 	public function ContentLocale() {
 		if($this->dataRecord && $this->dataRecord->hasExtension('Translatable')) {
```

Comment:
Changing this to `$this->dataRecord->hasField('Locale')` would mean being able to remove the hard-coded translatable dependency too.